### ISSUE TITLE
Caught exception in read(): 'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by sergey-dryabzhinsky** [sergey-dryabzhinsky/dedupsqlfs#98]
```
2017-05-21 05:05:17,751 - DedupFS - ERROR - Traceback (most recent call last):
  File "/opt/rusoft/dedupsqlfs/dedupsqlfs/fuse/operations.py", line 765, in read
    data = self.__get_block_data_by_offset(fh, offset, size)
  File "/opt/rusoft/dedupsqlfs/dedupsqlfs/fuse/operations.py", line 1480, in __get_block_data_by_offset
    block = self.__get_block_from_cache(inode, n + first_block_number)
  File "/opt/rusoft/dedupsqlfs/dedupsqlfs/fuse/operations.py", line 1399, in __get_block_from_cache
    self.getLogger().debug("-- db size: %s" % len(item["data"]))
TypeError: 'NoneType' object is not subscriptable
```

**Comment by sergey-dryabzhinsky** [comment]
Probably fixed in 1.2.921.
Need to check it.

**Comment by sergey-dryabzhinsky** [comment]
Nope. Maybe 1.2.922 fix it.

**Comment by sergey-dryabzhinsky** [comment]
Something wrong with cache reading/expiring/flushing.

**Comment by sergey-dryabzhinsky** [comment]
Something really wrong. Too much cache I think.

**Comment by sergey-dryabzhinsky** [comment]
It's all broken. Need to remove all cache code, recheck all data paths.

**Comment by sergey-dryabzhinsky** [comment]
Fixed in 1.9.923
